auth/src/main/res/values/colors.xml

```diff
@@ -1,9 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="fui_colorPrimary">#3F51B5</color>
```

Comment:
Since it's technically a "breaking change" to remove them and cause a build error for that 1%, let's keep the default colors in place.

In a future version we can really pull the rug out and make sure developers override them.